brayns/common/scene/Model.h

```diff
@@ -96,9 +96,15 @@ class ModelParams : public ModelInstance
     const std::string& getName() const { return _name; }
     void setPath(const std::string& path) { _updateValue(_path, path); }
     const std::string& getPath() const { return _path; }
+    void setLoaderName(const std::string& loaderName)
+    {
+        _updateValue(_loaderName, loaderName);
+    }
+    const std::string& getLoaderName() const { return _loaderName; }
```

Comment:
I think this wouldn't be needed following my previous suggestion. Eventually, GeometryParams is going to disappear and you'll have to revisit this change because the core won't have enough information to decide the loader name by itself, and in any case, it will depend in the plugins that are loaded.